CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   the test content datastream xml files under images/testcontent, and update the broken-content script
   to include that image tag.
 
+- The Compliance Operator now falls back to using the v1beta1 CronJob API
+  on clusters where the v1 CronJob API is not available which fixes a
+  [regression](https://issues.redhat.com/browse/OCPBUGS-2156) which was introduced
+  in the previous release (v0.1.56)
```

Comment:
Note for other reviews and a question.

This only impacted users deploying 0.1.56 on clusters that didn't have the Cron v1 API, correct?

Reply:
Yes, I think so.